python-avd/pyavd/_utils/default.py

```diff
@@ -1,19 +1,28 @@
 # Copyright (c) 2023-2024 Arista Networks, Inc.
 # Use of this source code is governed by the Apache License 2.0
 # that can be found in the LICENSE file.
-from typing import Any
+from __future__ import annotations
 
+from typing import TYPE_CHECKING
 
-def default(*values: Any) -> Any:
+if TYPE_CHECKING:
+    from typing import TypeVar
+
+    # TODO: Once we drop 3.10 support this can be imported from typing.
+    from typing_extensions import Unpack
+
+    T = TypeVar("T")
+
+
+def default(*values: Unpack[tuple[Unpack[tuple[T | None, ...]], T]]) -> T:
     """
     Accepts any number of arguments. Return the first value which is not None.
 
     Last resort is to return None.
 
-    Parameters
-    ----------
-    *values : any
-        One or more values to test
+    Args:
```

Comment:
we probably need to align all our docstrings and pick a rule - here we have a different style in the same docsrting between Args and Returns

Reply:
fixed